src/ctapipe/tools/process.py

```diff
@@ -307,6 +368,12 @@ def start(self):
                 )
                 continue
 
+            if not self.should_process(event):
```

Comment:
Do we really want this? It is a huge load of debug printouts. If you really want to have it, please consider changing log level to TRACE

Reply:
No strong opinion about the log level from my side.

Reply:
This log is removed because the event type filter is now used.